Sources/Widgets/Widgets3D/ResliceCursorWidget/test/testMultipleRotationPlanes.js

```diff
@@ -0,0 +1,365 @@
+import vtkWidgetManager from 'vtk.js/Sources/Widgets/Core/WidgetManager';
+
+import {
+  ViewTypes,
+  CaptureOn,
+} from 'vtk.js/Sources/Widgets/Core/WidgetManager/Constants';
+
+import { getViewPlaneNameFromViewType } from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers';
+
+/** Create js object that contains all needed parameters to define a vtkjs view
+  @param gc Garbage collector used when defined new object
+  @param {ViewTypes} viewType : Define the view type
+  @param {ResliceCursorWidget} widget: main widget where the new widget instance will be added
+*/
+function createView(gc, viewType, widget) {
+  const obj = {
+    container: gc.registerDOMElement(document.createElement('div')),
+    renderWindow: gc.registerResource(vtkRenderWindow.newInstance()),
+    renderer: gc.registerResource(vtkRenderer.newInstance()),
+    GLWindow: gc.registerResource(vtkOpenGLRenderWindow.newInstance()),
+    interactor: gc.registerResource(vtkRenderWindowInteractor.newInstance()),
+    widgetManager: gc.registerResource(vtkWidgetManager.newInstance()),
+    reslice: gc.registerResource(vtkImageReslice.newInstance()),
+    resliceMapper: gc.registerResource(vtkImageMapper.newInstance()),
+    resliceActor: gc.registerResource(vtkImageSlice.newInstance()),
+    origin: [],
+    point1: [],
+    point2: [],
+    viewType,
+  };
+
+  obj.renderer.getActiveCamera().setParallelProjection(true);
+  obj.renderer.setBackground([1, 1, 0]);
+  obj.renderWindow.addRenderer(obj.renderer);
+  obj.renderWindow.addView(obj.GLWindow);
+  obj.renderWindow.setInteractor(obj.interactor);
+  obj.GLWindow.setContainer(obj.container);
+  obj.GLWindow.setSize(400, 400);
+  obj.interactor.setView(obj.GLWindow);
+  obj.interactor.initialize();
+  obj.interactor.bindEvents(obj.container);
+  obj.widgetManager.setRenderer(obj.renderer);
+  obj.interactor.setInteractorStyle(
+    gc.registerResource(vtkInteractorStyleImage.newInstance())
+  );
+  obj.widgetInstance = obj.widgetManager.addWidget(widget, viewType);
+  obj.widgetManager.enablePicking();
+  // Use to update all renderers buffer when actors are moved
+  obj.widgetManager.setCaptureOn(CaptureOn.MOUSE_MOVE);
+
+  obj.reslice.setTransformInputSampling(false);
+  obj.reslice.setAutoCropOutput(true);
+  obj.reslice.setOutputDimensionality(2);
+  obj.resliceMapper.setInputConnection(obj.reslice.getOutputPort());
+  obj.resliceActor.setMapper(obj.resliceMapper);
+
+  return obj;
+}
+
+test('Test rendering when several rotations plane', (t) => {
+  const gc = testUtils.createGarbageCollector(t);
+  t.ok('Widgets', 'ResliceCursorWidgets rendering');
+
+  // --------------------------------------------------------------------------
+  // Create constants
+  // --------------------------------------------------------------------------
+  const viewTypes = [
+    ViewTypes.XZ_PLANE,
+    ViewTypes.YZ_PLANE,
+    ViewTypes.XY_PLANE,
```

Comment:
I think it would make sense to create some convenient constants in Constant.js:
```
const xyzToViewType = [ 
  ViewTypes.YZ_PLANE,
  ViewTypes.XZ_PLANE,
  ViewTypes.XY_PLANE
];
const viewTypeToXYZ = { 
  ViewTypes.YZ_PLANE: 0,
  ViewTypes.XZ_PLANE: 1,
  ViewTypes.XY_PLANE: 2
];
const viewTypeToViewName = { 
  ViewTypes.YZ_PLANE: "X",
  ViewTypes.XZ_PLANE: "Y"),
  ViewTypes.XY_PLANE: "Z"
];
const viewNameToViewType = { 
  "X": ViewTypes.YZ_PLANE,
  "Y": ViewTypes.XZ_PLANE,
  "Z": ViewTypes.XY_PLANE
];
```

Reply:
That's seems risky if anyone, for any reason, change the order of the xyzToViewType. And I don't understand the link between xyzToViewType and viewTypeToXYZ.

Reply:
There is no reason to change the order.

I do not understand what you do not understand :-)
```
xyzToViewType[0] ->  ViewTypes.YZ_PLANE
xyzToViewType[ViewTypes.YZ_PLANE] ->  0
```

Reply:
Why do we want to use indexes as it's only used in the example ?

Reply:
It is typically a convenient way to deal with axes.

It is actually used in both the example and the test. therefore it makes sense to expose it as conveniency